pkg/cloudprovider/providers/azure/azure.go

```diff
@@ -321,9 +326,20 @@ func NewCloud(configReader io.Reader) (cloudprovider.Interface, error) {
 		config.CloudProviderBackoffDuration = backoffDurationDefault
 	}
 
-	// Do not add master nodes to standard LB by default.
-	if config.ExcludeMasterFromStandardLB == nil {
-		config.ExcludeMasterFromStandardLB = &defaultExcludeMasterFromStandardLB
+	if strings.EqualFold(config.LoadBalancerSku, loadBalancerSkuStandard) {
+		// Do not add master nodes to standard LB by default.
+		if config.ExcludeMasterFromStandardLB == nil {
+			config.ExcludeMasterFromStandardLB = &defaultExcludeMasterFromStandardLB
```

Comment:
this fixed another issue?

Reply:
No, it's just moving the original condition within standard sku. ExcludeMasterFromStandardLB is only applied for standard LB.